apps/console/src/features/groups/components/wizard/group-basics.tsx

```diff
@@ -64,15 +68,17 @@ export const GroupBasics: FunctionComponent<GroupBasicProps> = (props: GroupBasi
     } = props;
 
     const { t } = useTranslation();
-    const dispatch = useDispatch();
+    const dispatch: Dispatch<any> = useDispatch();
```

Comment:
Can't we use the type from redux here?
`import { Dispatch } from "redux";`

Reply:
Fixed with b076d597fcdb4c9acc769a503b0fb20262ba0309